### PR TITLE
documents load image

### PR DESCRIPTION
**Pull request by mikebrow**
Address issue #638 by showing detailed steps for creating and loading images.

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: docs/crictl.md [hunk 1]
```diff
@@ -30,12 +30,45 @@ debug: true
 ```
 
 ## Download and Inspect a Container Image
+The pull command tells the container runtime to load a container image from a
```
Comment (Random-Liu) [inline docs/crictl.md]: s/load/download?

File: docs/crictl.md [hunk 2]
```diff
@@ -30,12 +30,45 @@ debug: true
...
 $ crictl pull busybox
   ...
 $ crictl inspecti busybox
   ... displays information about the image.
 ```
+
+## Directly Load a Container Image
+Another way to load an image into the container runtime is with the load
+command. With the load command you inject a container image into the container
+runtime from a file. First you need to create the container image. For example
```
Comment (Random-Liu) [inline docs/crictl.md]: container image tarball
or
container image file?

File: docs/crictl.md [hunk 3]
```diff
@@ -30,12 +30,45 @@ debug: true
...
+to create a pause container using Docker:
```
Comment (Random-Liu) [inline docs/crictl.md]: image tarball
or 
image file?

File: docs/crictl.md [hunk 4]
```diff
@@ -30,12 +30,45 @@ debug: true
...
+```console
+$ docker pull k8s.gcr.io/pause-amd64:3.1
+  3.1: Pulling from pause-amd64
+  67ddbfb20a22: Pull complete
+  Digest: sha256:59eec8837a4d942cc19a52b8c09ea75121acc38114a2c68b98983ce9356b8610
+  Status: Downloaded newer image for k8s.gcr.io/pause-amd64:3.1
+$ docker save k8s.gcr.io/pause-amd64:3.1 > pause.tar
```
Comment (Random-Liu) [inline docs/crictl.md]: nit: `-o`

File: docs/crictl.md (hunk 1 above)
Comment (mikebrow) [reply docs/crictl.md]: done

File: docs/crictl.md (hunk 2 above)
Comment (mikebrow) [reply docs/crictl.md]: done

File: docs/crictl.md (hunk 3 above)
Comment (mikebrow) [reply docs/crictl.md]: done

File: docs/crictl.md (hunk 2 above)
Comment (Random-Liu) [reply docs/crictl.md]: Not changed? :P

File: docs/crictl.md (hunk 3 above)
Comment (Random-Liu) [reply docs/crictl.md]: Not changed?

File: docs/crictl.md (hunk 4 above)
Comment (mikebrow) [reply docs/crictl.md]: yeah works both ways.. docker documents it both ways..

done

File: docs/crictl.md (hunk 3 above)
Comment (mikebrow) [reply docs/crictl.md]: refreshing..

**Pull request merged**